k2/csrc/fsa_utils.h

```diff
@@ -219,16 +219,17 @@ Array1<FloatType> GetForwardScores(FsaVec &fsas, Ragged<int32_t> &state_batches,
 /*
   Return array of total scores (one per FSA), e.g. could be interpreted as
   the data probability or partition function.
-         @param [in] fsas   Input Fsa or FsaVec (must have 2 or 3 axes)
+         @param [in] fsas   Input FsaVec (must have 3 axes)
```

Comment:
Can we build an FsaVec from the input FSA and do a recursive call?

Reply:
yeah, we can, but we don't need that version for now (If we do, we should do the same thing to GetForwardScores as well.)